src/__tests__/downshift.get-button-props.js

```diff
@@ -71,6 +71,27 @@ test('getToggleButtonProps returns all given props', () => {
   )
 })
 
+test('getToggleButtonProps accepts a mapStateToProps function', () => {
+  render(
```

Comment:
Could we rewrite this (and the other similar tests) to work more like the one in `downshift.get-input-props`?